### PR TITLE
Manager updates 

### PR DESCRIPTION
**Pull request by codenamejason**
Fixes: #3515 

Pleas note that this PR is already growing and want to keep to scope to the 3 pages referenced in the screenshots. We will follow up with more PR's to continue with the round-manager UI overhaul.

[Figma](https://www.figma.com/design/bB15jl5yBhuceBS3qgXngi/Manager-S20?node-id=775-19058&t=Y3mTHWKbxcEOvvi7-0)

## Description

Screenshot 2024-06-26 at 12 56 09 PM

Screenshot 2024-06-26 at 12 56 38 PM

Screenshot 2024-06-26 at 12 57 07 PM

Screenshot 2024-06-26 at 12 57 22 PM

Screenshot 2024-06-26 at 12 57 35 PM

Screenshot 2024-06-26 at 12 57 51 PM

## This PR:
🚨 all changes on `round-manager` application only.

- [x] Updates  landing page.
- [x] Updates program view page
- [x] Updates round view page which includes a partial applications redo to include Checker. We still have to coordinate with the Checker app our flow from `round-manager` to Checker and back.

### CODE REVIEW

File: packages/grant-explorer/src/features/round/ViewRoundPage.tsx [hunk 1]
```diff
@@ -384,6 +386,8 @@ function RoundPage(props: {
 
   const chain = getChainById(chainId);
 
+  console.log("round", round, "chainId", chainId, "roundId", roundId);
```
Comment (thelostone-mc) [inline packages/grant-explorer/src/features/round/ViewRoundPage.tsx:389]: ^ removeeee

File: packages/round-manager/src/features/common/ActionPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import { CheckIcon, ExclamationCircleIcon } from "@heroicons/react/solid";
+import { MdClose } from "react-icons/md";
+
+// todo: next scope: add actions to the action panel
+// const actions: any[] = [];
```
Comment (thelostone-mc) [inline packages/round-manager/src/features/common/ActionPanel.tsx:5]: ^ do we need to do this right now ?

File: packages/round-manager/src/features/common/Auth.tsx [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import { Outlet } from "react-router-dom";
 import { useAccount, usePublicClient, useWalletClient } from "wagmi";
 import { Spinner } from "./Spinner";
-import { ReactComponent as LandingBanner } from "../../assets/landing/banner.svg";
-import { ReactComponent as LandingLogo } from "../../assets/landing/logo.svg";
+// import { ReactComponent as LandingBanner } from "../../assets/landing/banner.svg";
```
Comment (0xKurt) [inline packages/round-manager/src/features/common/Auth.tsx:4]: remove comments

File: packages/round-manager/src/features/common/ActionPanel.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    <div className="mt-4 bg-blue-100 rounded-2xl">
+      <div className="flex flex-col my-4 ml-5">
+        <div className="flex flex-row items-center justify-start mt-4">
+          <ExclamationCircleIcon className="h-5 w-5 inline mr-2 text-grey-500" />
+          <span className="text-grey-500 font-normal font-sans">
+            Actions to be taken
+          </span>
+        </div>
+        <ul className="flex flex-row justify-start space-x-5">
+          {/* {actions &&
```
Comment (0xKurt) [inline packages/round-manager/src/features/common/ActionPanel.tsx:18]: remove comment

File: packages/round-manager/src/features/common/ActionPanel.tsx (hunk 2 above)
Comment (0xKurt) [inline packages/round-manager/src/features/common/ActionPanel.tsx:5]: remove

File: packages/round-manager/src/features/round/ApplicationStatusView.tsx [hunk 5]
```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { ApplicationStatusViewProps } from "../common/types";
+// import { Link } from "react-router-dom";
+// import { ReactComponent as CheckerLogo } from "../../assets/checker-logo.svg";
```
Comment (0xKurt) [inline packages/round-manager/src/features/round/ApplicationStatusView.tsx:4]: remove comments

File: packages/round-manager/src/features/round/ApplicationStatusView.tsx [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+            <p className="text-3xl">{allAppsCount}</p>
+            <p className="mt-4">Total applications</p>
+          </div>
+        </div>
+        {/* todo: add a check if any actions and display the action panel */}
+        {/* <ActionPanel /> */}
+        {/* todo: implement - need the checker URL for passing the right params */}
+        {/* <div className="flex justify-center mt-6 border border-grey-100 p-4 rounded-2xl">
+          <div className="flex flex-col justify-between items-center text-grey-500 font-normal">
+            <span className="flex flex-row items-center font-normal mb-4">
```
Comment (0xKurt) [inline packages/round-manager/src/features/round/ApplicationStatusView.tsx:45]: do we need this? else remove

File: packages/round-manager/src/test-utils.tsx [hunk 7]
```diff
@@ -43,10 +43,10 @@ import {
 } from "common";
 import { zeroAddress } from "viem";
 import { DistributionMatch } from "data-layer";
-import { WagmiProvider } from "wagmi";
-import queryClient, { config } from "./app/wagmi";
-import { RainbowKitProvider } from "@rainbow-me/rainbowkit";
-import { QueryClientProvider } from "@tanstack/react-query";
+// import { WagmiProvider } from "wagmi";
+// import queryClient, { config } from "./app/wagmi";
+// import { RainbowKitProvider } from "@rainbow-me/rainbowkit";
```
Comment (0xKurt) [inline packages/round-manager/src/test-utils.tsx:48]: remove comments

**Review by 0xKurt — APPROVED** [review]

**Review by melissa-neira — APPROVED** [review]
lgtm. will spin up a clean up ticket to address UI discrepancies

**Pull request merged**